docker-compose.yml

```diff
@@ -52,6 +54,15 @@ services:
       - postgres_data:/var/lib/postgresql/data/
     env_file:
       - ./.env
+  db-test:
+    image: postgres:12-alpine
+    ports:
+      - 5430:5432
+    volumes:
+      - postgres_data_test:/var/lib/postgresql/data/
+    env_file:
+      - ./.env
```

Comment:
nit: We can create a test DB in the same Postgres process in the `db` image. This would save some resources for someone running this on a slower computer. One way of doing this by copying an init script to `/docker-entrypoint-initdb.d` directory in the image. https://hub.docker.com/_/postgres

This can be updated in a later PR

Reply:
Good suggestion, added a ticket to our backlog 👍